docs/source/examples/simpleflight_output.txt

```diff
@@ -43,8 +43,6 @@ Sensitivities
      W_{W_{coeff1}} : 0.2903  Wing Weight Coefficent 1        
             N_{ult} : 0.2903  ultimate load factor            
      W_{W_{coeff2}} : 0.1303  Wing Weight Coefficent 2        
-             (CDA0) : 0.09156 fuselage drag area              
-                \mu : 0.08599 viscosity of air                
```

Comment:
I thought we dropped this change? (it's showing up in #856). should we rebase that PR?